common/internal_model/src/test/scala/uk/ac/wellcome/models/parse/PeriodParserTest.scala

```diff
@@ -348,6 +348,14 @@ class ParserTest extends AnyFunSpec with Matchers with Inspectors {
           "fl. 1999-2001 [gaps]"))
     }
 
+    it("strips Roman numerals") {
```

Comment:
nit: maybe this should be

```suggestion
    it("ignores Roman numerals") {
```

The word "strips" makes me expect it to be removed from the display string (more like whitespace). Yes, I might be splitting hairs.